### ISSUE TITLE
'prevents' not calling automat/mychem

### ISSUE THREAD

**Issue by cbizon** [ranking-agent/strider#67]
```
{'message': {'query_graph': {'nodes': {'a': {'category': 'biolink:ChemicalSubstance',
     'id': 'CHEBI:6067'},
    'b': {'category': 'biolink:Disease'}},
   'edges': {'ab': {'subject': 'a',
     'object': 'b',
     'predicate': 'biolink:prevents'}}}}}
```
No results from strider, logs show it is not calling automat/mychem which as some of this data

**Comment by cbizon** [comment]
Same problem for a predicate "contraindicated_for"

```
{'message': {'query_graph': {'nodes': {'a': {'category': 'biolink:ChemicalSubstance',
     'id': 'CHEBI:52172'},
    'b': {'category': 'biolink:Disease'}},
   'edges': {'ab': {'subject': 'a',
     'object': 'b',
     'predicate': 'biolink:contraindicated_for'}}}}}
```

returning no answers, not calling automat/mychem

**Comment by alongreyber** [comment]
Fixed by #81.
